opentech/apply/funds/templates/funds/applicationsubmission_simplified_detail.html

```diff
@@ -0,0 +1,66 @@
+{% extends "base-apply.html" %}
+
+{% block title %}{{ object.title }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+<div class="admin-bar">
+    <div class="admin-bar__inner">
+        <h1 class="beta heading heading--no-margin heading--bold">{{ object.title }}</h1>
+        <h5 class="heading heading--meta">
+            <span>{{ object.stage }}</span>
+            <span>{{ object.page }}</span>
+            <span>{{ object.round }}</span>
+            <span>Lead: {{ object.lead }}</span>
+            <span>
+                Project
+                <a class="link--transparent link--underlined"
+                   href="{{ object.project.get_absolute_url }}">
+                    {{ object.project }}
+                </a>
+            </span>
+        </h5>
+    </div>
+</div>
+
+<div class="wrapper wrapper--large wrapper--tabs js-tabs-content">
+    <div class="wrapper wrapper--sidebar">
+        <article class="wrapper--sidebar--inner">
+            <header class="heading heading--submission-meta heading-text zeta">
+                <span>Submitted: <strong>{{ object.submit_time.date }} by {{ object.user.get_full_name }}</strong></span>
+                <span>Last edited: <strong>{{ object.live_revision.timestamp.date }} by {{ object.live_revision.author }}</strong></span>
+            </header>
+
+            <h3>Proposal Information</h3>
+            <div class="grid grid--proposal-info">
+                <div>
+                    <h5>Requested Funding</h5>
+                    {{ object.get_value_display }}
+                </div>
+
+                <div>
+                    <h5>Project Duration</h5>
+                    {{ object.get_duration_display }}
+                </div>
+
+                <div>
+                    <h5>Legal Name</h5>
+                    {{ object.get_full_name_display }}{% if request.user.is_apply_staff %} (<a href="{% url 'wagtailusers_users:edit' object.user_id %}">Edit account</a>){% endif %}
+                </div>
+
+                <div>
+                    <h5>Email</h5>
+                    {{ object.get_email_display }}
+                </div>
+                <div class="grid__cell--span-two">
+                    <h5>Address</h5>
+                    {{ object.get_address_display }}
```

Comment:
So, the slack message was perhaps unlear. You removed the bit we needed to keep and kept the bit i'm not really interested in. 

Expected output was
```django
<div class="rich-text rich-text--answers">
    {{ object.output_text_answers }}
</div>
```
where output text answers is a new method 
```python
def output_text_answers(self):
    return mark_safe(''.join(self.render_text_blocks_answers()))
```
located near here https://github.com/OpenTechFund/opentech.fund/blob/767de79cc46550939055576e7c554054e1d3eabc/opentech/apply/funds/models/mixins.py#L230

Reply:
Fixed and rebased.